docs/logging/README.md

```diff
@@ -0,0 +1,108 @@
+# Minio Logging Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+This document explains how to configure Minio server to log to different logging targets.
+
+## Targets
+Minio supports currently two target types
+
+- console
```

Comment:
Should we talk about `Audit Target` here?

Reply:
Audit target is a separate section @kannappanr

Reply:
Then it should be called `Log Targets`

Reply:
Done